app/views/notes/show.html.erb

```diff
@@ -130,3 +141,51 @@
   </div>
 
 </div><!--/span-->
+
+
+<style type="text/css">
+  .hexbin-hexagon {
+    stroke: #000;
+    stroke-width: 1px;
+  }
+</style>
+
+<script type="text/javascript">
```

Comment:
This should be in its own javascript file in `/assets/javascripts/location-display.js` or something similar. Thanks!